Jenkinsfile

```diff
@@ -413,7 +413,8 @@ pipeline {
                         unstash 'MathSetup'
                         script {
                             sh "echo O=0 > make/local"
-                            sh "echo CXX=${CLANG_CXX} -Werror >> make/local"
+                            sh "echo CXX=${GCC} -Werror >> make/local"
+                            sh "echo CXXFLAGS+=-fsanitize=address >> make/local"
```

Comment:
If you intend to keep this we will want to study what it does to the overall test time. It looks like it is currently at 3x the normal time for this stage and counting.

If you _don't_ want to keep this and it's just a sanity check, you might also want to remove the `O=0` line above -- my understanding is sanitizers won't catch everything at -O0

Reply:
I just wanted to run this via jenkins to confirm this fixed the error. Let me set this to a higher optim level